cheroot/server.py

```diff
@@ -1146,6 +1151,10 @@ class HTTPServer(object):
     nodelay = True
     """If True (the default since 3.1), sets the TCP_NODELAY socket option."""
 
+    # issue #17
```

Comment:
please add such comments in the end of the line they refer to in order to avoid lines decoupling over time